client/i18n/locales/english/intro.json

```
@@ -7256,12 +7256,204 @@
+      "zh-a1-module-initials-and-finals": "Initials and Finals",
+      "zh-a1-module-greetings-and-basic-introductions": "Greetings and Basic Introductions",
+      "zh-a1-module-asking-and-giving-basic-information": "Asking and Giving Basic Information",
+      "zh-a1-module-introducing-others": "Introducing Others",
+      "zh-a1-module-getting-to-know-the-team": "Getting to Know the Team",
+      "zh-a1-module-talking-about-personal-skills": "Talking about Personal Skills",
+      "zh-a1-module-discussing-team-skills": "Discussing Team Skills"
+    },
+    "module-intros": {
+      "zh-a1-module-introduction-and-certification-overview": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-initials-and-finals": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-greetings-and-basic-introductions": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-asking-and-giving-basic-information": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-introducing-others": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-getting-to-know-the-team": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-talking-about-personal-skills": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      },
+      "zh-a1-module-discussing-team-skills": {
+        "note": "Coming Winter 2025",
+        "intro": ["PLACEHOLDER: Write the module intro here."]
+      }
+    },
     "blocks": {
-      "learn-essential-courtesies-at-the-office": {
-        "title": "Learn Essential Courtesies at the Office",
-        "intro": [
-          "Learn simple and polite ways to communicate at work. In this module, you'll learn essential phrases for greeting colleagues, expressing thanks, apologizing, and saying goodbye."
-        ]
+      "zh-a1-learn-certification-introduction": {
+        "title": "Certification Introduction",
+        "intro": ["", ""]
+      },
+      "zh-a1-learn-single-finals": {
+        "title": "Single Finals",
+        "intro": ["", ""]
+      },
+      "zh-a1-learn-initials": { "title": "Initials", "intro": ["", ""] },
+      "zh-a1-learn-compound-finals": {
+        "title": "Compound Finals",
+        "intro": ["", ""]
+      },
+      "zh-a1-learn-nasal-finals": {
+        "title": "Nasal Finals",
+        "intro": ["", ""]
+      },
+      "zh-a1-review-the-pinyin-review": {
+        "title": "The Pinyin Review",
```

Should this be just

```suggestion
      "zh-a1-review-pinyin": {
        "title": "Pinyin Review",
```

(We'll need to update the block folder and the dashedName in `a1-professional-chinese.json` if we apply this change.)